message FinancePB{
    optional int32 coin = 1;
    optional int32 gold = 2;
    optional int32 hero_soul = 3;
}
